_opensearch/rest-api/create-index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: Create index
+parent: REST API reference
+grand_parent: OpenSearch
+nav_order: 7
+---
+
+# Create index
+
+While you can create an index by using a document as a base, you can also just create an empty index for use later.
+
+## Example
+
+The following example demonstrates how to create an index with specifies numbers of shards and replicas, specifies that `age` is of type `integer`, and assigns a `sample-alias1` alias to the index.
+
+```json
+PUT /sample-index1
+
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 2,
+      "number_of_replicas": 1
+    }
+  },
+  "mappings": {
+    "properties": {
+      "age": {
+        "type": "integer"
+      }
+    }
+  },
+  "aliases": {
+    "sample-alias1": {}
+  }
+}
+```
+
+## Index naming restrictions
+
+OpenSearch Indices have the following naming restrictions:
+
+- All letters must be lowercase.
+- Index names can't begin with underscores (`_`) or hyphens (`-`).
+- Index names can't contain spaces, commas, or the following characters:
+
+  `:`, `"`, `*`, `+`, `/`, `\`, `|`, `?`, `#`, `>`, or `<`
+
+## URL parameters
+
+You can include the following URL parameters in your request. All parameters are optional.
+
+Parameter | Type | Description
+:--- | :--- | :---
+include_type_name | Boolean | If `true`, the request expects a type in the body of mappings. Default is `false`.
```

Comment:
Given that types have been removed/deprecated, should we mark this as something you really shouldn't change to true? As far as I know, everything should have a type of just plain ol' `_doc`.